README.md

```diff
@@ -376,6 +376,36 @@ To disable the splashscreen add the following preference to `config.xml`:
 <preference name="SplashScreenDelay" value="0"/>
 ```
 
+##### Display app version (Android, iOS)
+
+To display your app version on splashscreen add the following preference to `config.xml`:
+```xml
+<preference name="ShowSplashScreenAppVersion" value="true" />
+```
+
+Now just configure your app version into the widget version attribute, like that : 
+```xml
+<widget version="1.0" >...</widget>
+```
+
+
+###### Prefix app version
+To set a prefix version add the following preference to `config.xml`:
+```xml
+<preference name="SplashScreenAppVersionPrefix" value="v" />
+```
+In this case you'll have `v1.0`
+
+
+###### Customize app version
+To customise the version color, size and horizontal position add the following preference to `config.xml`:
```

Comment:
... the color, size and horizontal position of your version add ...
preference => preferences